app/views/transition_landing_page/_page_header.html.erb

```diff
@@ -40,7 +40,7 @@
       <div class="govuk-grid-row">
         <div class="govuk-grid-column-full">
           <div class="landing-page__header-arrow-container">
-            <%= image_tag 'white-arrow.svg', class: "landing-page__header-arrow" %>
+            <%= image_tag 'white-arrow.svg', class: "landing-page__header-arrow", alt: "" %>
```

Comment:
nice i didn't think ot check those for alt tags before
@injms recently [mentioned an aria tag](https://gds.slack.com/archives/CARGH33JS/p1594629172069500?thread_ts=1594629065.068800&cid=CARGH33JS) that stops these being read out by screen readers, do we need those too?

Reply:
I don't think an `aria-hidden` attribute is needed - the blank `alt` text should mean that this isn't read aloud by screenreader.